fairlearn/postprocessing/__init__.py

```diff
@@ -7,10 +7,12 @@
 learn how to adjust the predictor's output from the training data.
 """
 
+from ._interpolated_thresholder import InterpolatedThresholder  # noqa: F401
 from ._threshold_optimizer import ThresholdOptimizer  # noqa: F401
 from ._plotting import plot_threshold_optimizer  # noqa: F401
 
 __all__ = [
-    "ThresholdOptimizer",
-    "plot_threshold_optimizer"
+    "InterpolatedThresholder",
```

Comment:
You'll also want to export `ThresholdOperation`. That's another structured object, where we should review whether we like its current API enough to make it external.